docs/shared/tutorials/react-standalone.md

```diff
@@ -392,22 +479,133 @@ Nx allows you to separate this logic into "local libraries". The main benefits i
 - better scalability in CI by enabling independent test/lint/build commands for each library
 - better scalability in your teams by allowing different teams to work on separate libraries
 
-### Creating Local Libraries
+### Create a Local Library
 
-Let's assume our domain areas include `products`, `orders` and some more generic design system components, called `ui`. We can generate a new library for each of these areas using the React library generator:
+Let's assume our domain areas include `products`, `orders` and some more generic design system components, called `ui`. We can generate a new library for these areas using the React library generator:
 
 ```
 nx g @nx/react:library products --unitTestRunner=vitest --bundler=none --directory=modules/products
+```
+
+Note how we use the `--directory` flag to place the library into a subfolder. You can choose whatever folder structure you like to organize your libraries.
+
+Nx tries to set up your workspace to work with the modular library architecture, but depending on your existing configuration, you may need to tweak some settings. In this repo, you'll need to do a few things in order to prepare for future steps.
```

Comment:
Reword without "tries", maybe just "Nx sets up your workspace...", since saying "tries" feels a bit weak on what Nx does.

Reply:
Yep, I'll remove "tries" once we fix all the issues